src/Chapter1.hs

```diff
@@ -632,7 +642,11 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = a + b
+  where
+    a = quot (mod (abs n) 100) 10
+    b = mod (abs n) 10
```

Comment:
Usually `quot` is paired with `rem`, while `mod` with `div`, these pair are comprehensive to each other 🙂